### DISCUSSION TITLE
Error adding plugin "An error occurred while installing bigdecimal (3.1.8), and Bundler cannot"

### DISCUSSION THREAD

**Discussion by anuragpathak21** [docker-library/redmine#330] [General]
While installing pro plugins getting below error 

 An error occurred while installing bigdecimal (3.1.8), and Bundler cannot continue.

<img width="847" alt="image" src="https://github.com/docker-library/redmine/assets/50142570/80ea8b25-b124-4c66-ba72-83b4dbeb51c3">

<img width="742" alt="image" src="https://github.com/docker-library/redmine/assets/50142570/80074cc3-5f16-402d-8486-6afc23e2aa9d">

<img width="887" alt="image" src="https://github.com/docker-library/redmine/assets/50142570/a211cbdf-87e5-4bba-b37d-cfa09eb6f2dd">

**image: redmine:5.1.2**

**Plugins added**
redmine_agile-1_6_9-pro.zip
redmine_budgets-1_0_5-pro.zip
redmine_checklists-3_1_23-pro.zip
redmine_contacts_helpdesk-4_2_2-pro.zip
redmine_contacts_invoices-4_2_9-pro.zip
redmine_crm-4_4_1-pro.zip
redmine_crm_mailchimp-1_0_4.zip
redmine_drive-1_2_1-pro.zip
redmine_finance-2_1_9-pro.zip
redmine_people-1_6_6-pro.zip
redmine_products-2_2_2-pro.zip
redmine_questions-1_0_5-pro.zip
redmine_reporter-2_0_4-pro.zip
redmine_resources-2_0_1-pro.zip
redmine_zenedit-2_0_6-pro.zip

**Comment by anuragpathak21** [comment]
@tianon Why do you convert all the issues as discussion without providing any comments/explanation. I reported this as error with proper details and you moved it to discussion. If issues cannot be reported please disable issues from the repository so that people like me will save time reporting the issues.

**Reply by tianon** [reply]
Sorry - to clarify, issues are for issues in the image itself that we might fix in the image.  Issues installing plugins do not typically fall into that category, so I move them to discussions because support with the issue is going to be best-effort and community-driven.

You're probably more likely to receive help with this if you can post a simple/small _reproducer_ (some way to use the image as-is, with no additional parts, and still produce the errors you're running into).

In your case, it looks like the plugins you're installing require additional Ruby packages which need to compile from source and that's failing -- you need to read the referenced log file to find out _what_ is failing to compile and which dependencies are missing if you want to make progress.

**Comment by darkhallow1378** [comment]
I have same issue. Any comment on this?

**Reply by darkhallow1378** [reply]
@anuragpathak21 , do you solve this problem finially?

**Reply by TrestenPool** [reply]
Any update on this @darkhallow1378 ?

**Reply by LaurentGoderre** [reply]
Seems like it is a duplicate of #328
